StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableView.swift

```diff
@@ -255,6 +242,14 @@ final class InstitutionTableView: UIView {
         }
     }
     
+    func setTableHeaderView(_ tableHeaderView: UIView?) {
+        if let tableHeaderView = tableHeaderView {
```

Comment:
nit: not sure if there's a swift version compatibility concern but can you use the shorthand? `if let tableHeaderView {`

Reply:
Changed it in the new commit! 

I think I am just stuck being "old school." The syntax feels weird because it looks like a "floating" 'uninitialized' variable:

```
let variable1: Int

// vs.

if let variable2 {

}
```

@bric-stripe out of curiosity, if you had to convince someone why `if let variable2` is better than `if let variable2 = variable2`, what would you say? The old way definitely looks worse/unnecessary when directly compared that way, but there is a question of cost in terms of readability/'learnability' (its another instance of "magic shorthand syntax" that, seemingly, only makes sense if you know what its a shorthand of)

Reply:
> if you had to convince someone why if let variable2 is better than if let variable2 = variable2, what would you say?

I don't feel strongly here but I think its one of the easier to infer bits of syntax so would just go for the fewer words option